boards/nxp/frdm_mcxn236/frdm_mcxn236.yaml

```diff
@@ -0,0 +1,25 @@
+#
+# Copyright 2024 NXP
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+identifier: frdm_mcxn236
+name: NXP FRDM MCXN236
+type: mcu
+arch: arm
+ram: 192
```

Comment:
It looks like the `sram0` bank we are using for RAM is 256KB, right? So we should make this value 256 as well

Reply:
Updated.